src/Uno.UI.RemoteControl.Server.Processors/HotReload/MetadataUpdates/CompilationWorkspaceProvider.cs

```diff
@@ -24,31 +25,40 @@ internal static class CompilationWorkspaceProvider
 			Dictionary<string, string> properties,
 			CancellationToken ct)
 		{
-			if (properties.TryGetValue("UnoEnCLogPath", out var EnCLogPath))
+			if (properties.TryGetValue("UnoHotReloadDiagnosticsLogPath", out var logPath))
 			{
 				// Sets Roslyn's environment variable for troubleshooting HR, see:
 				// https://github.com/dotnet/roslyn/blob/fc6e0c25277ff440ca7ded842ac60278ee6c9695/src/Features/Core/Portable/EditAndContinue/EditAndContinueService.cs#L72
-				Environment.SetEnvironmentVariable("Microsoft_CodeAnalysis_EditAndContinue_LogDir", EnCLogPath);
-			}
+				Environment.SetEnvironmentVariable("Microsoft_CodeAnalysis_EditAndContinue_LogDir", logPath);
 
-			var globalProperties = new Dictionary<string, string> {
-				// Mark this compilation as hot-reload capable, so generators can act accordingly
-				{ "IsHotReloadHost", "True" },
-			};
+				Environment.SetEnvironmentVariable("MSBUILDDEBUGENGINE", "1");
```

Comment:
// Unconditionally enable binlog generation in msbuild. See https://github.com/dotnet/project-system/blob/4210ce79cfd35154dbd858f056bfb9101f290e69/docs/design-time-builds.md?L61